internal/pkg/config/v1/kafka_cluster_config.go

```diff
@@ -41,3 +41,12 @@ func (k *KafkaClusterConfig) EncryptAPIKeys() error {
 	}
 	return nil
 }
+
+func (k *KafkaClusterConfig) GetApiSecret() string {
+	if k.APIKeys != nil {
```

Comment:
Just to be extra safe, maybe we can also check `k != nil` first.

Reply:
```suggestion
	if k != nil && k.APIKeys != nil {
```